src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeActionService.java

```diff
@@ -87,6 +88,27 @@ public void moveEnvelopeToCompleted(UUID envelopeId) {
         log.info("Envelope {} status chaged to COMPLETED", envelope.getZipFileName());
     }
 
+    @Transactional
+    public void moveEnvelopeToAborted(UUID envelopeId) {
+        Envelope envelope = envelopeRepository.findById(envelopeId)
```

Comment:
do we need this method? it's not being used

Reply:
it will be called from a controller